libs/ui/editor/src/lib/Atoms/blockList.tsx

```diff
@@ -91,7 +91,7 @@ export interface BlockListValue<T extends string = string, V = any> {
   value: V
 }
 
-export type BlockMap = Record<string, BlockCaseProps>
+type BlockMap = Record<string, BlockCaseProps>
```

Comment:
Shouldn't this be exported?

Reply:
so this is one of the cases where there are multiple things with the same name exported from different files. The same thing is being exported from atoms. there's an error reported in barrel file. Solution for that is to not export one of the values with the same name